landscape/client/configuration.py

```diff
@@ -473,7 +480,9 @@ def query_landscape_edition(self):
             self.landscape_domain = self.prompt_get_input(
                 "Landscape Domain: ",
                 True,
-            )
+            ).strip('/')
+            self.landscape_domain = re.sub(r'^https?://', '',
+                                           self.landscape_domain)
```

Comment:
I notice you are using single quotes instead of double quotes like we have elsewhere in the code. Our linters don't catch this, but it would be good to be consistent.

Reply:
Totally forgot about this. Done